frontend/src/components/common/dokumente/Dokumente.vue

```diff
@@ -38,7 +38,7 @@
               block
               color="secondary"
               elevation="1"
-              :disabled="currentNumberOfAddedFiles >= maxNumberOfFiles"
+              :disabled="!addDokumentButtonEnabled"
```

Comment:
Das Icon zum Löschen der eines Dokuments in der Komponente `DokumentenListe` ist auch zu disablen, da es sich um eine Manipulation der Abfrage handelt.

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/e45d8aed-9325-4141-b23b-3102af76937a)

Evtl. zusätzlich die beiden Icons zum Löschen und zum Download eines Dokuments mit einem `v-btn` des Typs `icon` umschließen und das Clockevent in den Button auslagern. 

```ts
                 <v-btn
                  :id="'"'dokumente_listitem_' + index + '_loeschen_button'""
                  icon
                  @click="deleteDokument(item)"
                >
                  <v-icon> mdi-delete</v-icon>
                </v-btn>
```

Reply:
Erledigt.
Button um Icon... Das Lehrgeld haben wir beide anscheinend (bitter) bezahlt :-)